main.nf

```diff
@@ -33,13 +35,27 @@ WorkflowMain.initialise(workflow, params, log)
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 */
 
-include { PROTEINFOLD } from './workflows/proteinfold'
+if (params.mode == "alphafold2") {
+    include { ALPHAFOLD2 } from './workflows/alphafold2'
+} else if (params.mode == "colabfold") {
+    include { COLABFOLD } from './workflows/colabfold'
+}
```

Comment:
Did you do that to avoid the empty process in the terminal?

Reply:
If I am not wrong, when I omit the if-else statement and include both workflows, I get a warning for unused workflow and modules

Reply:
In the way it is structured now, this won't work, since the ALPHAFOLD2 and COLABFOLD have to be initiated inside the main workflow NFCORE_PROTEINFOLD. What's the problem with the existing structure?